src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,95 @@ history:
         version: '1'
 
 ---
-There are 3 steps shown by 3 tabs to brand your application:
+{{! excerpt}}
+The branding tab manages the display of the login page of an application.
+{{! /excerpt}}
 
-![]({{file name='branding_tabs.png'}} ?w=310,h=42)
+![]({{file name='branding-studio-menu.png'}} ?w=600,border=true)
 
-*   The [**Main Pages**](#mainp) tab will let you manage the global rendering of your application
-*   The [**Login Page**](#login) tab will let you manage the login page rendering
-*   The [**Custom CSS**](#css) will let you define your CSS rules that will be applied on each page of Nuxeo
+To create a customized login page:
 
-{{> anchor 'mainpages'}}
+1. In the **Configuration** menu, go to the **Branding** tab.
```

Comment:
Menu not tab! Tabs are used to navigate inside the screen, otherwise you may get people confused.